src/frontend/src/pages/new-challenge/new-challenge.tsx

```diff
@@ -1,7 +1,150 @@
+                                    </CardContent>
+                                    <CardActions>
+                                        {/* Create an 'onClick' method that calls the API */}
+                                        <Button size="small" color="primary">
+                                            Change
+                                </Button>
+                                    </CardActions>
+                                </Card>
+                            </Grid>
+                            <Grid item xs={4}>
+                                <Card>
+                                    <CardContent>
+                                        <Typography gutterBottom>
+                                            Optional Definition2
+                                </Typography>
+                                    </CardContent>
+                                    <CardActions>
+                                        {/* Create an 'onClick' method that calls the API */}
+                                        <Button size="small" color="primary">
+                                            Change
+                                </Button>
+                                    </CardActions>
+                                </Card>
+                            </Grid>
+                        </Grid>
+                    </Box>
+                    {/*  spacing */}
+                    {/*<div style={{ padding: 50 }}></div>*/}
+                    <Box p={2}>
+                        <Typography gutterBottom variant="h4">
+                            3 - Input your sentence
+                </Typography>
+                    </Box>
+                    {/*<div style={{ padding: 50 }}>*/}
+
+                    <div>
+                        <Box p={2}>
+                            <TextField
+                                fullWidth
+                                multiline
+                                rows="4"
+                                variant="outlined"
+                            />
+                        </Box>
+                    </div>
+
+                    {/*<div style={{ padding: 50 }}></div>*/}
+                    <Box p={2}>
+                        <Typography gutterBottom variant="h4">
+                            4 - Send to some friends
+                </Typography>
+                    </Box>
+                    <Autocomplete
+                        multiple
+                        options={friends}
+                        getOptionLabel={friends => friends}
+                        renderInput={params => (
+                            <TextField
+                                {...params}
+                                variant="standard"
+                                label="friends"
+                                placeholder="friend"
+                            />
+                        )}
+                    ></Autocomplete>
```

Comment:
Please change the Autocomplete component to be just a single tag. 
```suggestion
                    />
```

Reply:
This issue has been addressed.